pilot/pkg/features/pilot.go

```diff
@@ -347,6 +347,12 @@ var (
 		"The timeout to send the XDS configuration to proxies. After this timeout is reached, Pilot will discard that push.",
 	).Get()
 
+	RemoteClusterTimeout = env.RegisterDurationVar(
+		"PILOT_REMOTE_CLUSTER_TIMEOUT",
+		30*time.Second,
```

Comment:
Seems far too high - should be 2-3 times the normal sync time. If a cluster is down - 30 sec on each istiod startup is too much. Maybe 10 seconds ?

Reply:
I'll try to measure the time from server start -> ready and emit a metric and then use 2-3x that value.

Reply:
It can be really high on a big cluster

Reply:
Since we're not sure - can we add a metric for the 'sync time', with the cluster name as one dimension ? Would also help detect other problems. Can be in a separate PR - I'm ok with merging this with 30 sec for 1.9, better than forever.